docs/data/joy/customization/dark-mode/DarkModeByDefault.tsx

```diff
@@ -4,26 +4,28 @@ import Sheet from '@mui/joy/Sheet';
 import Chip from '@mui/joy/Chip';
 import Typography from '@mui/joy/Typography';
 
-const theme = extendTheme({ cssVarPrefix: 'demo' });
+const theme = extendTheme({
+  cssVarPrefix: 'demo',
+  defaultColorScheme: 'dark',
+  cssRule: '.demo_%s-mode-by-default',
+});
 
 export default function DarkModeByDefault() {
   return (
     <CssVarsProvider
-      defaultMode="dark"
       // the props below are specific to this demo,
       // you might not need them in your app.
       //
       theme={theme}
-      // the selector to apply CSS theme variables stylesheet.
-      colorSchemeSelector="#demo_dark-mode-by-default"
+      colorSchemeNode={null}
```

Comment:
What's `colorSchemeNode` use?

Reply:
This is for the demo to prevent overriding the `<html>` data attribute because the root Doc has `CssVarsProvider` too.

Reply:
I removed it, not necessary to use.